docs/integrations/google_colab.rst

```diff
@@ -3,38 +3,108 @@
 
-1. Install Neptune client:
+Introduction
+------------
 
-    Go to your first cell in Google Colab and install `neptune-client`:
+This guide will show you how to:
 
-    .. code-block:: Bash
+#. Install ``neptune-client``
+#. Connect Neptune to your notebook and create the first experiment
+#. Log simple metrics to Neptune and explore them in the UI
 
-        pip install neptune-client
+Before you start
+----------------
 
-2. Set Neptune API token:
+Make sure that you have an account with both `Google <https://support.google.com/accounts/answer/27441?hl=en>`_ and `Neptune <https://neptune.ai/register>`_.
 
-    Go to the Neptune web app and `get your API token <../python-api/how-to/organize.html#find-my-neptune-api-token>`_. Set it to the environment variable `NEPTUNE_API_TOKEN`:
+Quickstart
+----------
 
-    .. code-block:: Bash
+|Run on Colab|
 
-        env NEPTUNE_API_TOKEN='your_private_neptune_api_token=='
+Step 1: Install Neptune client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Go to your first cell in Google Colab and install ``neptune-client``:
 
-3. Delete this cell.
+.. code-block:: Bash
 
-    .. warning::
+   pip install neptune-client
 
-        It is very important that you delete this cell, so as not to share your private token with anyone.
+Step 2: Set Neptune API token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-4. Now, run your training script with Neptune:
+#. Go to the Neptune web app and `get your API token <../python-api/how-to/organize.html#find-my-neptune-api-token>`_
 
-    .. code-block:: Python
+   .. image:: https://neptune.ai/wp-content/uploads/get_token.gif
+      :target: https://neptune.ai/wp-content/uploads/get_token.gif
+      :alt: Getting your Neptune API token
+   
+#. Run the code below:
+    
+   .. code-block:: Python
 
-        import neptune
-        neptune.init('USER_NAME/PROJECT_NAME')
+      from getpass import getpass
+      api_token = getpass('Enter your private Neptune API token')
 
-        with neptune.create_experiment():
-            neptune.send_metric('auc', 0.92)
\ No newline at end of file
+#. Enter the token in the input box. This will save your token to ``api_token``
+
+Step 3: Initialize your Neptune Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below:
+
+.. code-block:: Python
+
+    import neptune
+
+    neptune.init(project_qualified_name = 'your_user_name/your_project_name'
+                 , api_token = api_token)
+
+Step 4: Run your training script with Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below:
+
+.. code-block:: Python
+
+    from numpy import random
+    from time import sleep
+
+    with neptune.create_experiment():
```

Comment:
I don't think every reader uses/understands the `with` structure. 
I'd simply go with `neptune.create_experiment()` and `neptune.stop()` at the end.